semcore/side-panel/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [3.41.0] - 2024-07-29
+
+### Changed
+
+- Close icon to `Button` component.
```

Comment:
I see change of color only, maybe it's better to mention that?

Reply:
Yes, fixed